scipy/stats/_stats_py.py

```diff
@@ -7917,15 +7808,14 @@ def _attempt_exact_2kssamp(n1, n2, g, d, alternative):
                     jrange = np.arange(h)
                     prob = np.prod((n1 - jrange) / (n1 + jrange + 1.0))
                 else:
-                    with np.errstate(over='raise'):
-                        num_paths = _count_paths_outside_method(n1, n2, g, h)
+                    num_paths = _count_paths_outside_method(n1, n2, g, h)
```

Comment:
```suggestion
                    with np.errstate(over='raise'):
                        num_paths = _count_paths_outside_method(n1, n2, g, h)
```
Probably should keep this. I don't think it's necessary, because other checks will cause this function to return False if `num_paths` is `inf` or `nan`, but for the builds that don't use Pythran, it provides an early exit if something goes wrong in `_count_paths_outside_method`.